public/javascripts/src/quiz.js

```diff
@@ -228,10 +253,15 @@ var activeCopyButton = function(){
   $(this).text('Copied').addClass('copied');
 }
 
-function getSpanColor(score){
+function getSpanColor(score, possible_score){
+  var intScore = parseInt(possible_score);
   if (league == 'nfl'){
-    score = score / 5
-  }else{score = score/3 }
+    score = score / 5;
+  }
+  else if (intScore <= 10){
```

Comment:
Don't we need to do this case first, for football goats/leaders which will only have 10?